src/components/LoginPage/FormLayout.tsx

```diff
@@ -0,0 +1,69 @@
+import { styled } from 'styled-components';
+
+const Form = styled.form`
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	width: 450px;
+	height: 750px;
+	border-radius: 10px;
+	background-color: rgba(0, 0, 0, 0.637);
+	overflow: hidden;
+
+	fieldset {
+		position: relative;
+		margin: 48px 68px;
+
+		legend {
+			font-size: 3.2rem;
+			font-family: Helvetica, sans-serif;
+			font-weight: bold;
+			color: #ffffff;
+			margin-bottom: 1em;
+		}
+	}
+
+	div {
```

Comment:
Tak nie robimy w styled components. To prosta droga aby "pokusić się" o wprowadzenie klas, albo aby doprowadzić do problemów. Wyobraź sobie, że stylujesz po h1, h2, p itd a nagle zmieniasz tag w JSX... style przestaną działać. Każdy element stylowany musi być osobną zmienną w styled components.

Konwencja jest taka że zaczynasz nazwę zmiennej od "Styled" a później w komponencie robisz "import * as S from './styled' (Tak zwykle nazywamy te pliki) wtedy mozesz uzywac <S.StyledForm> itd